regulations/templates/regulations/base.html

```diff
@@ -44,8 +46,6 @@
         <!--[if lt IE 9]>
             <script src="{%static "regulations/js/built/lib/respond/respond.min.js" %}"></script>
         <![endif]-->
-        {% block preload %}
-        {% endblock %}
```

Comment:
Yep. BTW, are we exposing this block for downstream applications to use? Doesn't look like anything else in -site uses `preload`.